src/database.ts

```diff
@@ -142,21 +201,16 @@ class Database extends ServiceObject {
       parent: instance,
       id: name,
       methods,
-      createMethod: (_, options, callback) => {
+      createMethod: (_: {}, options: spanner_client.spanner.v1.ISession |
+        CreateSessionCallback, callback: CreateSessionCallback) => {
         return instance.createDatabase(formattedName_, options, callback);
       },
     } as {}) as ServiceObjectConfig);
 
+    this.pool_ = typeof poolOptions === 'function' ? new (poolOptions as Any)(this, null) : new SessionPool(this, poolOptions);
```

Comment:
after using `CreateDatabaseOptions` and `CreateDatabaseCallback` this should not need `Any`
```suggestion
    this.pool_ = typeof poolOptions === 'function' ? new poolOptions(this, null) : new SessionPool(this, poolOptions);
```

Reply:
added `CreateDatabaseOptions` and `CreateDatabaseCallback`, but still it gives error so keep as it is.

Reply:
Tested by me, no errors. Maybe try one more time.

Reply:
have you set `noImplicitAny` to `true`? I have tried it today as well it gives compilation error.